README.md

```diff
@@ -73,36 +73,39 @@ and processor architecture listed below.
 
 ### Supported operating system versions
 
+* [CentOS Linux] 7
 * [Debian] 11
 * [Docker]
 * [FreeBSD] 13.0
-* [Ubuntu] 20.04
-* [Rocky Linux] 8.4
+* [Ubuntu] 22.04
+* [Rocky Linux] 8.6
 
 Only the latest long-term supported version of Debian, FreeBSD, Rocky Linux and
-Ubuntu, respectively, is supported.
+Ubuntu, respectively, is supported. Support for CentOS Linux 7 will be dropped
+by Zonemaster release v2023.1.
 
 Only the Docker images provided by the Zonemaster project on [Docker Hub] are
 supported. Currently only Zonemaster-CLI is supported on Docker. Docker itself
 can run on any of the [Docker] supported OSs (Linux, MacOS and Windows).
 
 [Rocky Linux] has replaced CentOS in Zonemaster version v2021.2 since CentOS 8
-will not be supported beyond 2021-12-31 and CentOS 7 is old and does not support
-modern OpenSSL required by Zonemaster. Rocky Linux is also a Red Hat derivative
-and is available at large cloud providers.
+is not supported anymore and CentOS 7 is old and does not support modern OpenSSL
+required by Zonemaster. Rocky Linux is also a Red Hat derivative and is available
+at large cloud providers.
 
 ### Supported database engine versions
 
-Operating System | MariaDB | PostgreSQL | SQLite
----------------- | --------| -----------|------------------------
-Debian 11        | 10.5    | 13.3       | 3.34
-Docker           | n/a     | n/a        | n/a
-FreeBSD 13.0     | 5.7     | 13.5       | 3.35.5
-Ubuntu 20.04     | 10.3    | 12.4       | 3.31.1
-Rocky Linux 8.4  | 10.3    | 10.17      | 3.26
+Operating System | MariaDB | PostgreSQL      | SQLite
+---------------- | --------| ----------------|------------------------
+CentOS Linux 7   | 5.5     | *not supported* | 3.36.0
+Debian 11        | 10.5    | 13.3            | 3.34
```

Comment:
For Debian (v11.3), I am getting PostgreSQL v13.7. MariaDB is v10.5.15. Also, regarding SQLite, note that while v3.34.1 is the version shipped with this version of Debian, Zonemaster::Backend uses v3.32.3 (which is the version bundled in DBD::SQLite v1.66. See [here](https://metacpan.org/dist/DBD-SQLite/changes#L57-61)).

Reply:
We agreed back in time to have just two "levels" in the version number for MariaDB/MySQL and PostgreSQL, but then we should do that for SQLite too. I now changed all to x.y instead of some x.y.z.